action.yml

```diff
@@ -36,6 +36,13 @@ inputs:
     required: false
     default: false
 
+  create output:
```

Comment:
Is `create output` the best name for this feature? `as json` perhaps? If you're happy with it as it is, then fair enough

Reply:
In theory it already creates the JSON output (when this is not `True` it will just be output an empty `dict`/JSON object). So in my mind this would toggle whether the JSON object would have content or not.

Reply:
Fair enough!